Documentation/compatibility/wpf-TextBox-PasswordBox-text-selection-does-not-follow-system-colors.md

```diff
@@ -0,0 +1,40 @@
+## WPF TextBox/PasswordBox Text Selection Does Not Follow System Colors
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+In .NET Framework 4.7.1 and earlier versions, WPF `System.Windows.Controls.TextBox` and `System.Windows.Controls.PasswordBox` could only render a text selection in the Adorner layer.
+In some system themes this would occlude text, making it hard to read.  In .NET Framework 4.7.2 and later, developers have an option of enabling a non-Adorner based selection rendering 
+scheme that alleviates this issue.
+
+- [X] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+A developer who wants to utilize this change must set the following AppContext flag appropriately.  
```

Comment:
What is the default value? If it's false for 4.7.2 and later, then there's no need for the switch.

Reply:
This is always true, there is no version quirk.  Developers must always opt into this at this point.